src/controller/securemgr/verifier/verifier.go

```diff
@@ -26,58 +26,50 @@ import (
 
 // cwl - Container White List
 const (
-	cwlFileName			= "containerwhitelist.txt"
+	cwlFileName = "containerwhitelist.txt"
 
-	ERROR_NONE			= "ERROR_NONE"
-	INVALID_PARAMETER	= "INVALID_PARAMETER"
-	SECUREMGR_ERROR		= "INTERNAL_SECUREMGR_ERROR"
-	NOT_ALLOWED_COMMAND	= "NOT_ALLOWED_COMMAND"
+	ERROR_NONE          = "ERROR_NONE"
+	INVALID_PARAMETER   = "INVALID_PARAMETER"
+	SECUREMGR_ERROR     = "INTERNAL_SECUREMGR_ERROR"
+	NOT_ALLOWED_COMMAND = "NOT_ALLOWED_COMMAND"
 )
 
 // VerifierImpl structure
 type VerifierImpl struct{}
 
 var (
-	containerWhiteList	[]string
-	logPrefix			= "[securemgr: verifier]"
-	verifierIns			*VerifierImpl
-	initialized			= false
-	cwlFilePath			= ""
+	containerWhiteList []string
+	logPrefix          = "[securemgr: verifier]"
+	verifierIns        *VerifierImpl
+	initialized        = false
+	cwlFilePath        = ""
 )
 
 type RequestDescInfo struct {
 	//ContainerName string
 	ContainerHash string
 }
 
-type RequestSecureMgr struct {
-	SecureInsName	string
-	CmdType			string
-	Desc			[]RequestDescInfo
+type RequestVerifierConf struct {
+	SecureInsName string
+	CmdType       string
+	Desc          []RequestDescInfo
 }
 
-type ResponseSecureMgr struct {
-	Message			string
-	SecureCmpName	string
+type ResponseVerifierConf struct {
+	Message       string
+	SecureCmpName string
 }
 
-// SecureMgrExternalAPI is the interface implemented by external REST API
-type SecureMgrExternalAPI interface {
-	RequestSecureMgr(containerInfo RequestSecureMgr) ResponseSecureMgr
+// VerifierConf is the interface implemented by external REST API
```

Comment:
@tdrozdovsky @suresh-lc Any impact about this API change? It is just the call name change though, but it should be clear that : 1) previous SecureMgrExternalAPI itself is the API provided by SecureMgr, now 2) the proposed commit changes it to VerifierConf, which consists of one of the workflows (functions) in the territory of SecureMgr.

Reply:
The good and correct remark, but I think we should not worry about this since the external api has not changed. This is only the internal changes that relate to the handler (I decided to simplify our code and not introduce a separate api).
Regarding adding a token: this in any case will lead to the fact that any request will require adjustment. Unfortunately, protection always leads to difficulties in configuration and use. Especially if it has not been laid since the beginning of development. thanks for the comment

Reply:
@tdrozdovsky Thanks for your comment. Overall I agree with your point. By the way, for things more clarified, is there any API document regarding this? It would be helpful for us to fully understand your opinion in the code level. Thank you in advance.

Reply:
Did you see the [doc/secure_manager.md](https://github.com/tdrozdovsky/edge-home-orchestration-go/blob/external-rest-api-authentication/doc/secure_manager.md) or you need more detailed information. Later I plan to add information to  `edge_orchestration_api.yaml`.

Reply:
@tdrozdovsky Yes. Better to present a bit more details like yaml files. But understood that everything comes at one time. Thank you.

Reply:
@MoonkiHong @suresh-lc I corrected some inaccuracies in the description of the quick start (changes were made after the commit on February 18). And also **added REST API descriptions** for secure mode (**YAML format**)